universal-application-tool-0.0.1/app/views/AzureFileUploadViewStrategy.java

```diff
@@ -105,7 +105,7 @@ public Tag renderFileUploadBlockSubmitForms(
     return div(formTag, skipForms, buttons)
         .withId("azure-upload-form-component")
         .with(
-            footer(viewUtils.makeWebJarsTag(WebJarJsPaths.AZURE_STORAGE_BLOB)),
```

Comment:
Looks good. Is it worth creating a separate issue tracking reverting to this path once the Azure issue is fixed upstream? We could then add a TODO against it here.

Reply:
Done.